packages/loopring_v3/contracts/aux/access/LoopringIOExchangeOwner.sol

```diff
@@ -135,7 +155,31 @@ contract LoopringIOExchangeOwner is SelectorBasedAccessManager, ChiDiscount, ERC
         // Process the callback logic.
         _beforeBlockSubmission(blocks, config);
 
+        // Do flash deposits
+        for (uint i = 0; i < flashConfig.deposits.length; i++) {
+            IExchangeV3(target).flashDeposit(
```

Comment:
Should we change `flashDeposit` to a plural form to reduce inter-contract calls?

Reply:
Yep, working on some changes now.